### PR TITLE
refactor: add new programs and accounts to cli test-validator

### PR DESCRIPTION
**Pull request by ananas-block**
### Changes:
- added new programs and accounts to cli test-validator command

### Notes:
- xtask didn't work out unfortunately because I couldn't just run the solana program test in a non sbf env
- thus I just put it into a test and modified the getAccountState.sh script

### To run:
- programs need to be built
``anchor build``
- in cli
``pnpm build``
``./cli/test_bin/run test-validator``

### CODE REVIEW

File: cli/accounts/governance_authority_pda_3MtrKu5Mjgh3JqeE5PeRzw2Ld28XjFgbbph67E6UERSx.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"pubkey":"3MtrKu5Mjgh3JqeE5PeRzw2Ld28XjFgbbph67E6UERSx","account":{"lamports":1670400,"data":["92V2ansKL5F9ineLIqDCqQnhiwC5EFF0HDBN+u+Yy9c0sxQG63l/Mf8AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==","base64"],"owner":"5WzvRtu7LABotw1SUEpguJiKU27LRGsiCnF5FH6VV7yP","executable":false,"rentEpoch":18446744073709551615,"space":112}}
```
Comment (vadorovsky) [inline cli/accounts/governance_authority_pda_3MtrKu5Mjgh3JqeE5PeRzw2Ld28XjFgbbph67E6UERSx.json:1]: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAaaaaaaaaaaaaaaaaaaaaaaa

File: programs/compressed-pda/tests/test.rs [hunk 2]
```diff
@@ -136,3 +136,36 @@ async fn test_create_execute_compressed_transaction_2() {
     let res = create_and_send_transaction(&mut context, &[instruction], &payer).await;
     assert!(res.is_err());
 }
+use solana_cli_output::CliAccount;
+use tokio::fs::write as async_write;
+#[ignore = "this is a helper function to regenerate accounts"]
+#[tokio::test]
+async fn regnerate_accounts() {
```
Comment (vadorovsky) [inline programs/compressed-pda/tests/test.rs:143]:
```suggestion
async fn regenerate_accounts() {
```

**Review by vadorovsky — COMMENTED** [review]
I like generating accounts through a test! Left some minor comments.

File: cli/accounts/governance_authority_pda_3MtrKu5Mjgh3JqeE5PeRzw2Ld28XjFgbbph67E6UERSx.json (hunk 1 above)
Comment (ananas-block) [reply cli/accounts/governance_authority_pda_3MtrKu5Mjgh3JqeE5PeRzw2Ld28XjFgbbph67E6UERSx.json:1]: AAAAAAAAAAAAAAAAAAAAA

**Review by vadorovsky — APPROVED** [review]
looks good, feel free to merge once green

File: scripts/getAccountState.sh [hunk 3]
```diff
@@ -1,77 +1,10 @@
...
-    solana account $value --output-file "cli/accounts/${key}.json" --output "json"
-done
+# run this to regenerate the following accounts:
+# merkle_tree_pubkey
+# indexed_array_pubkey
+# governance_authority_pda
+# group_pda
+# 
+# to add more accounts to regenerate, add them to setup_test_programs_with_accounts and test script
+cd programs/compressed-pda && cargo test-sbf regnerate_accounts -- --ignored && cd -
```
Comment (vadorovsky) [inline scripts/getAccountState.sh:10]: You need to fix the typo here as well

```suggestion
cd programs/compressed-pda && cargo test-sbf regenerate_accounts -- --ignored && cd -
```

**Review by vadorovsky — CHANGES_REQUESTED** [review]

File: scripts/getAccountState.sh (hunk 3 above)
Comment (ananas-block) [reply scripts/getAccountState.sh]: fixed

**Review by vadorovsky — APPROVED** [review]

**Pull request merged**